internal/resources/lokilogpipeline/resources_test.go

```diff
@@ -0,0 +1,20 @@
+package lokilogpipeline
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMakeLokiLogPipeline(t *testing.T) {
+	lokiLogPipeline := MakeLokiLogPipeline()
+
+	require.NotNil(t, lokiLogPipeline)
+	require.Equal(t, lokiLogPipeline.APIVersion, "telemetry.kyma-project.io/v1alpha1")
```

Comment:
require.Equal wants the expected value first, then the actual one. Does not make a difference in the test execution, but gives potentially a nicer output in the case of failures